spec/features/vote_article_spec.rb

```diff
@@ -18,28 +18,27 @@
       
       it "let's you upvote on the article", js: true do
         visit root_path
-        page.find("#vote-button-up-#{article.id}").click
+        page.find("#vote-Article-button-up-#{article.id}").click
         within('.article-vote') do
           expect(find('.article-vote-count')).to have_content(1);
         end
       end
       
       it "does't let you add multiple votes on the article", js: true do
         visit root_path
-        page.find("#vote-button-up-#{article.id}").click
-        page.find("#vote-button-up-#{article.id}").click
+        page.find("#vote-Article-button-up-#{article.id}").click
+        page.find("#vote-Article-button-up-#{article.id}").click
         within('.article-vote') do
           expect(find('.article-vote-count')).to have_content(1);
         end
```

Comment:
This is a good test.
What do you think of checking the database for the count increment as well?